creme/feature_extraction/agg.py

```diff
@@ -56,16 +62,43 @@ class Agg(base.Transformer):
         {'revenue_mean_by_place': 20.0}
         {'revenue_mean_by_place': 50.0}
 
+        # You can add lag to ``feature_extraction.Agg`` by setting the parameter ``shift``.
+        # The feature extration pipeline will shift the input serie with respect to the
+        # aggregates.
+
+        >>> agg = feature_extraction.Agg(
+        ...     on='revenue',
+        ...     by='place',
+        ...     how=stats.Mean(),
+        ...     shift=1,
+        ...     missing=0,
+        ... )
+
+        >>> for x in X:
+        ...     print(agg.fit_one(x).transform_one(x))
+        {'revenue_mean_shift_1_by_place': 0}
+        {'revenue_mean_shift_1_by_place': 0}
+        {'revenue_mean_shift_1_by_place': 16.0}
+        {'revenue_mean_shift_1_by_place': 42.0}
+        {'revenue_mean_shift_1_by_place': 20.0}
+        {'revenue_mean_shift_1_by_place': 50.0}
+
     References:
         1. [Streaming groupbys in pandas for big datasets](https://maxhalford.github.io/blog/streaming-groupbys-in-pandas-for-big-datasets/)
 
     """
 
-    def __init__(self, on: str, by: str, how: stats.Univariate):
+    def __init__(self, on: str, by: str, how: stats.Univariate, shift:int = 0, missing: float=0.):
```

Comment:
You don't to declare the type of `shift`, it's inferred from the default value AFAIK. Same goes for the missing parameter.